extensions/amp-subscriptions/0.1/test/test-amp-subscriptions.js

```diff
@@ -44,6 +46,21 @@ describes.realWin('amp-subscriptions', {amp: true}, env => {
     expect(initializeStub).to.be.calledOnce;
   });
 
+  it('should setup store and page on start', async() => {
```

Comment:
afaik, we do not allow `async` in tests or code. In tests though, you can use `yield`.

Reply:
Done